README.md

```diff
@@ -1,8 +1,8 @@
-# resinos
+# balenaOS
 
 Links:
 
-* [Feature request / Report a bug](https://github.com/balena-os/balenaos/issues)
+* [Feature request / Report a bug](https://github.com/balena-os/balena-os/issues)
 * [Documentation](https://balena.io/os/)
 * [meta-balena](https://github.com/balena-os/meta-balena) - *meta-balena is a Yocto layer used to build balena operating system images*
-* [Contribute to balenaos.io website](/CONTRIBUTING.md)
+* [Contribute to balena.io/os website](/CONTRIBUTING.md)
```

Comment:
On a side-note, should http://balenaos.io be setup to redirect to https://www.balena.io/os/, instead of just showing a "This domain name has been registered with Gandi.net" placeholder?
(like the way that http://etcher.io redirects to https://www.balena.io/etcher/ )

Reply:
@lurch thanks for pointing this out - will rectify!